Microsoft Defender/microsoft-defender-for-endpoints/_meta/smart-descriptions.json

```
@@ -21,6 +21,21 @@
       }
     ]
   },
+  {
+    "value": "{microsoft.defender.alert.title}",
+    "conditions": [
+      {
+        "field": "microsoft.defender.alert.title"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "service.name",
+        "target": "threat.technique.name",
+        "type": "detected attack"
+      }
```

```suggestion
    "value": "New alert detected {microsoft.defender.alert.title} affecting {service.name}",
    "conditions": [
      {
        "field": "microsoft.defender.alert.title"
      }
      {
        "field": "threat.technique.name"
      }
      {
        "field": "detected attack"
      }
    ],
    "relationships": [
      {
        "source": "service.name",
        "target": "threat.technique.name",
        "type": "detected attack"
      }
```

The smart description is meant to help user and be nice to read. I am afraid that ` "{microsoft.defender.alert.title}"` would be to simplistic.

---

I agree with @pve that the smart-description may be too simplistic. its suggestion also embedded the alert title with a different wording.
However, @pve, there is an error in your suggestion: you try to test the field "detected attack", which is not a field. I think you wanted to suggest to test "service.name"

---

Thanks @squioc, I fixed my suggestion !